src/plugins/visualizations/public/embeddable/visualize_embeddable.ts

```diff
@@ -377,29 +377,8 @@ export class VisualizeEmbeddable extends Embeddable<VisualizeInput, VisualizeOut
   };
 
   public supportedTriggers() {
-    // TODO: Report a correct list of triggers for each vis_type.
-    switch (this.vis.type.name) {
-      case 'area':
-      case 'heatmap':
-      case 'histogram':
-      case 'horizontal_bar':
-      case 'line':
-      case 'pie':
-      case 'table':
-      case 'tagcloud':
-        return [VIS_EVENT_TO_TRIGGER.filter];
```

Comment:
Here we only have supportedTriggers only for filter and not for brush. On each of the above visTypes I have also added brush where I think that it is supported. Not 100% sure though